src/Api/Email.php

```diff
@@ -46,4 +46,19 @@ public function send(EmailData $mail): array
 
       return $this->client->httpRequest($path, $body, $headers);
   }
+
+  /**
+   * Send a request to the UniOne API.
+   * @param  array                $body the request body
+   * @return array                the response with the status code
+   * @throws GuzzleException
+   * @throws BadResponseException
+   * @throws TransferException
+   */
+  public function subscribe(array $body): array
```

Comment:
Let's do the following:
1) Describe expected array keys in the method description;
2) Add a link to documentation: https://docs.unione.io/en/web-api-ref#email-send
3) Verify incoming array using assert package: https://github.com/webmozarts/assert (package should be added to composer.json);